test/parallel/test-timers-linked-list.js

```diff
@@ -103,3 +103,8 @@ assert.equal(C, L.shift(list));
 // list
 assert.ok(L.isEmpty(list));
 
+const list2 = L.create();
+const list3 = L.create();
+assert.ok(L.isEmpty(list2));
+assert.ok(L.isEmpty(list3));
+assert.ok(list2 != list3);
```

Comment:
the assertion failed... which makes sense in hindsight, circular lists point to themselves,
ie will not be the same.  The `isEqual` test above checks equivalence, ie that the circular
links are set up correctly.